youtube_dl/extractor/mytaratata.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class MyTaratataIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?mytaratata\.com/taratata/(?P<id>[a-zA-Z0-9_\-/]+)'
+    _TEST = {
+        'url': 'http://mytaratata.com/taratata/519/shaka-ponk-camille-et-julie-berthollet-smells-like-teen-spirit-nirvana',
+        'md5': '99657330eb7dec6d63a329d7f26ec93e',
+        'info_dict': {
+            'id': '7174',
+            'ext': 'mp4',
+            'title': 'TARATATA N°519 - Shaka Ponk / Camille et Julie Berthollet "Smells Like Teen Spirit" (Nirvana)',
+            'uploader': 'Taratata',
+            'description': 'Shaka Ponk / Camille et Julie Berthollet "Smells Like Teen Spirit" (Nirvana)',
+            'thumbnail': 're:https?://.*\.jpeg$',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # The title contains only the program name and episode number.
+        # Each episode containts many videos
+        title = self._og_search_title(webpage)
+        # The description is the title of the video within the episode.
+        description = self._og_search_description(webpage)
+
+        formats = []
+
+        video_source_re = re.compile(
+            r'<div [^>]*class="jwplayer" [^\>]*data-source="(?P<url>https?://.*/Taratata[^"]+\.mp4)"'
```

Comment:
Just like I expected storage location has changed meanwhile so this code is completely broken. As I've already pointed out: match **only** `data-source` attribute.

Reply:
will do